ios/MusicControlManager.m

```diff
@@ -188,6 +188,14 @@ - (void) toggleHandler:(MPRemoteCommand *) command withSelector:(SEL) selector e
     command.enabled = enabled;
 }
 
+- (void) listenToAudioRouteChange:(BOOL) enabled {
```

Comment:
I think it is safe IMO. If we run into multiple subscription it means that there is another problem somewhere else. 

Otherwise we can addObserver on init and removeObserver on dealloc. This way no need to worry about multiple trigger. what do you think ?